### DISCUSSION TITLE
dificulti

### DISCUSSION THREAD

**Discussion by Hosseinkrn** [Chia-Network/chia-blockchain#8850] [Support]
hi guys , i was trasfers some chia to my chia wallet and i didnt recived it becouse my wallet dont sync , now i tryng to sync my wallet for recive my chia coins and yesterday dificulti was 17 but today dificulti is 1248 and its so bad for me becouse my cpu is i3 10100f and its very low cpu and i cant sync my wallet , please help me how i can sync my wallet 

![Screenshot (154)](https://user-images.githubusercontent.com/92608238/137621851-637edc6a-1ce2-4f7a-8a82-e06bb2dca70f.png)

**Comment by BrandtH22** [comment]
@Hosseinkrn , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
